bahmnicore-omod/src/main/resources/config.xml

```diff
@@ -33,6 +33,7 @@
         <require_module>org.openmrs.module.episodes</require_module>
         <require_module>org.openmrs.module.auditlog</require_module>
         <require_module>org.bahmni.module.bahmnicommons</require_module>
+        <require_module>org.bahmni.module.fhir-ts-services</require_module>
```

Comment:
its probably better to use camelcasing "org.bahmni.module.fhirTsServices". If it means changing the package name in the other Omod, I think you should do so

Reply:
@angshu instead of using camelcasing for `org.bahmni.module.fhirTsServices`, can we use `org.bahmni.module.fhirtsservices` or `org.bahmni.module.ts.fhir.services` or simply `org.bahmni.module.fhirterminologyservices` ?

Oracle documentation says **Package names are written in all lower case to avoid conflict with the names of classes or interfaces.**
https://docs.oracle.com/javase/tutorial/java/package/namingpkgs.html
https://google.github.io/styleguide/javaguide.html#s5.2.1-package-names

cc: @binduak